lua/weapons/lucian_door_opener_wep.lua

```diff
@@ -116,32 +143,29 @@ function SWEP:SpawnLucian()
 				phys:EnableMotion(true)
 			end
 			
-			self:Remove()
-		
 		end
 		
+		self.Owner:ConCommand("lastinv")
```

Comment:
I did it this way because when using the old code, it sometimes did not select the previous weapon, and when it did, there was the holstered weapon animation played. This way it works as it should